src/main/java/duck/operation/Command.java

```diff
@@ -0,0 +1,148 @@
+package duck.operation;
+
+import duck.task.Deadline;
+import duck.task.Event;
+import duck.task.TaskList;
+import duck.task.Todo;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+public class Command {
+    private String command;
+    private String description;
+
+    /**
+     * Initialize the Command object
+     * @param command the command getting from ui.
+     * @param description the description of command,includes things to do, time and so on.
+     * @return Lateral location.
+     */
+    public Command(String command, String description) {
+        this.command = command;
+        this.description = description;
+    }
+
+    /**
+     * execute the command
+     * @param tasks the task list used to  record various tasks
+     * @param ui deals with interactions with the user
+     * @param storage deals with loading tasks from the file and saving tasks in the file
+     * @throws IOException
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws IOException {
+        String[] descriptionSplit;
+        switch (command) {
+        case "bye":
+        case "list":
+        case "hello":
+            ui.showCommandReply(command, tasks);
+            break;
```

Comment:
I like how you let the switch cases fall through for commands requiring the same output. Something most people forget about.